src/components/Tabs/tabs.scss

```diff
@@ -66,17 +71,12 @@
   }
 
   &--has-icon {
-    padding: 9px 14px 9px 10px;
+    padding: 9px 0;
 
     span {
       margin-left: 5px;
     }
   }
-
-  &--disabled {
-    cursor: not-allowed;
-    opacity: 0.5;
-  }
 }
```

Comment:
Hi Lisi, we need this style for when the **isDisabled** property is passed in SbTab.